src/utils/successResponse.js

```diff
@@ -1,3 +1,3 @@
-export default (res, status, body) => {
-  res.status(status).json(body);
+export default (res, status, message, body) => {
+  res.status(status).json({ message, body });
```

Comment:
Make sure you're carefully thorough with all the handlers depending on this change. The social auth controllers, for example, are among those and you seem to have missed them.